pkgs/applications/editors/vis/default.nix

```diff
@@ -1,8 +1,11 @@
 { lib, stdenv, fetchFromGitHub, pkg-config, makeWrapper, makeDesktopItem
-, ncurses, libtermkey, lpeg, lua
+, ncurses, libtermkey, lua
 , acl ? null, libselinux ? null
```

Comment:
```suggestion
, acl, libselinux
```

Reply:
can't it be null if not on linux ? doesn't look related to the PR.